youwol/pipelines/pipeline_typescript_weback_npm/regular/skeleton.py

```diff
@@ -156,7 +156,6 @@ def generate_readme(working_path: Path, input_template: Template):
 
 def generate_test_files(working_path: Path):
     path_dir_tests = working_path / 'src' / 'tests'
-    path_dir_tests.mkdir(parents=True)
```

Comment:
This line prevent some errors, for instance when running 'yarn build' in cdn-explorer with this branch of py-youwol we have the following error :
```
yarn run v1.22.19
$ yarn build:dev
$ yarn pre-build && webpack --mode development
$ yarn clean && yarn auto-gen
$ del-cli dist
$ python template.py
Found imported packages: {'@youwol/os-top-banner', '@youwol/http-clients', '@youwol/os-core', '@youwol/flux-view', '@youwol/cdn-client', '@youwol/fv-tabs', 'rxjs', 'rxjs/operators', '@youwol/installers-youwol'}
Traceback (most recent call last):
  File "/tmp/tmp.y3LaFb9dW2/cdn-explorer/template.py", line 52, in <module>
    generate_template(template)
  File "/tmp/tmp.y3LaFb9dW2/py-youwol/youwol/pipelines/pipeline_typescript_weback_npm/regular/skeleton.py", line 71, in generate_template
    generate_test_files(working_path=working_path)
  File "/tmp/tmp.y3LaFb9dW2/py-youwol/youwol/pipelines/pipeline_typescript_weback_npm/regular/skeleton.py", line 159, in generate_test_files
    f = open(path_dir_tests / 'fake.test.ts', 'w')
FileNotFoundError: [Errno 2] No such file or directory: '/tmp/tmp.y3LaFb9dW2/cdn-explorer/.template/src/tests/fake.test.ts'

```

Reply:
Good catch, an empty directory was missing in the repo. 
This is working:
rm -rf /tmp/test-cdn-explorer
log_prefix="===> "

cd /tmp
mkdir test-cdn-explorer
cd test-cdn-explorer

git clone https://github.com/youwol/py-youwol.git
cd py-youwol
git checkout feature/pipeline-ts-webpack
python3.9 -m venv venv
. venv/bin/activate
pip install -r dev-requirements.txt
echo "/tmp/test-cdn-explorer/py-youwol" >> ./venv/lib/python3.9/site-packages/py-youwol.pth

cd ..
git clone https://github.com/youwol/cdn-explorer.git
cd cdn-explorer
yarn
yarn build